packages/panels/tailwind.config.preset.js

```diff
@@ -2,7 +2,9 @@ import defaultPreset from '../support/tailwind.config.preset.js'
 import defaultTheme from 'tailwindcss/defaultTheme'
 
 defaultPreset.theme.extend.fontFamily = {
+    mono: ['var(--mono-font-family)', ...defaultTheme.fontFamily.mono],
     sans: ['var(--font-family)', ...defaultTheme.fontFamily.sans],
```

Comment:
@danharrin, shall we rename the regular font to `sans-font-family` here just like the others?

Reply:
no because I feel like it should feel clearly like the default

Reply:
Sure, fine by me